src/workerd/server/server.c++

```diff
@@ -2099,6 +2099,28 @@ class Server::WorkerService final: public Service,
             .attach(kj::addRef(*this));
       }
 
+      void abort(kj::Maybe<const kj::Exception&> reason) {
```

Comment:
If the actor is now "broken", but we're cancelling `onBrokenTask` and dropping `actor`, so `monitorOnBroken()` will just get canceled, then wouldn't we not remove the entry from `actors`? Should we `parent.actors.erase(key)` here?

Reply:
As of this PR, the only caller of `abort()` is `ActorNamespace::abortAll()`, which also clears the map immediately after, so it would be redundant for `abort()` to remove itself -- and actually problematic, as it would disrupt the iteration through the map in `abortAll()`.

That said, I think I need to go back and review this in my subsequent PRs, where I do start using abort in other cases, so thanks for the observation.